inc/search.class.php

```diff
@@ -5395,8 +5578,10 @@ static function displayConfigItem($itemtype, $ID, $data = []) {
     * @param string  $orig_itemtype   Original itemtype, used for union_search_type
     *
     * @return string String to print
+    *
+    * @since 10.0.0 Method is no longer static
    **/
-   static function giveItem($itemtype, $ID, array $data, $meta = 0,
+   public function giveItem($itemtype, $ID, array $data, $meta = 0,
```

Comment:
1. Should be mentionned in changelog.

2. This method is only privately used, may be changed to private.